spec.html

```diff
@@ -17107,9 +17107,8 @@ <h2>Syntax</h2>
     <emu-grammar type="definition">
       IfStatement[Yield, Await, Return] :
         `if` `(` Expression[+In, ?Yield, ?Await] `)` Statement[?Yield, ?Await, ?Return] `else` Statement[?Yield, ?Await, ?Return]
-        `if` `(` Expression[+In, ?Yield, ?Await] `)` Statement[?Yield, ?Await, ?Return]
+        `if` `(` Expression[+In, ?Yield, ?Await] `)` Statement[?Yield, ?Await, ?Return] [lookahead &ne; `else`]
     </emu-grammar>
-    <p>Each `else` for which the choice of associated `if` is ambiguous shall be associated with the nearest possible `if` that would otherwise have no corresponding `else`.</p>
```

Comment:
I'd like to see this paragraph converted to a note saying that the "lookahead &ne; `else`" has the effect of always associating an `else` to the nearest `if`, by only allowing `if-else` inside the then branch of an `if-else`.

Reply:
I wouldn't call it the "then" branch. I'd prefer the consequent, or the truthy branch, or the first branch.

Reply:
Sure, I don't have much of an opinion on the name.

Reply:
I added a note based on your comments with the term "truthy branch" as follows:

- Note that the &ldquo;[lookahead &ne; `else`]&rdquo; has the effect of always associating an `else` to the nearest `if`, by only allowing `if`-`else` inside the truthy branch of an `if`-`else`.